content/en/_headers

```diff
@@ -1,5 +1,5 @@
 /*
-    Content-Security-Policy: default-src 'none'; style-src 'self' 'unsafe-inline'  https://fonts.googleapis.com; script-src 'self' https://player.vimeo.com https://netlify-cdp-loader.netlify.app; img-src 'self' data:; frame-src 'self' https://player.vimeo.com https://outreach.abetterinternet.org; font-src 'self' https://fonts.gstatic.com data:; media-src 'self'; object-src 'self';
+    Content-Security-Policy: default-src 'none'; style-src 'self' 'unsafe-inline'; script-src 'self' https://player.vimeo.com https://netlify-cdp-loader.netlify.app; img-src 'self' data:; frame-src 'self' https://outreach.abetterinternet.org; font-src 'self' data:; media-src 'self'; object-src 'self';
```

Comment:
Is `https://player.vimeo.com` still needed here?

Reply:
Good catch, thanks!